tests/test_test.go

```diff
@@ -0,0 +1,64 @@
+package tests
```

Comment:
I think this file should be given a meaningful name, a long with an explanation for the tests package.